packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -641,6 +667,7 @@ export class SceneLoader {
             rootUrl: rootUrl,
             name: name,
             file: file,
+            rawData: rawData,

```

Comment:
nit (and i know the previous lines are doing the same thing):
```suggestion
            rawData,
```